build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker build --tag elifesciences/loris .
```

Comment:
usually we build `elifesciences/loris:${IMAGE_TAG}`

Reply:
went with `${IMAGE_TAG:-latest}` - this makes it usable without environment variables